packages/webview/dist/esm/index.js

```diff
@@ -0,0 +1,7 @@
+import { registerPlugin } from '@capacitor/core';
+const WebviewBackground = registerPlugin('WebviewBackground', {
+    web: () => import('./web').then((m) => new m.WebviewBackgroundWeb()),
+});
+export * from './definitions';
+export { WebviewBackground };
+//# sourceMappingURL=index.js.map
```

Comment:
I would strongly prefer not to include `dist` files in the repository, and to include building this package as part of building the main project. There are a variety of reasons (repository bloat, merge conflicts more likely, built code more difficult to review for security issues, difficulty verifying that the build matches the source, etc.) for this, but they're relatively unimportant for this particular package due to its small size. I'll defer to @raineorshine who [previously indicated](https://github.com/cybersemics/em/pull/2594#issuecomment-2564403642) that this is probably fine 👍

Reply:
That was my intuition as well, so I'm glad you said something. I know that @msdewitt did this to simplify the `webview` package release process, but I would love to discuss the feasibility of building it from source within the root project's build pipeline.